### PR TITLE
Automated Test - COMMERCE-6089 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
https://issues.liferay.com/browse/COMMERCE-6089, cc: @Pietro69

**Comment by kristinonias** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase [hunk 1]
```diff
@@ -57,12 +57,82 @@ definition {
...
+
+		CommerceEntry.gotoMenuTab(menuTab = "Visibility");
+
+		Click(
+			key_toggleSwitch = "wiwk___toggle-switch",
+			locator1 = "ToggleSwitch#SPECIFIC_TOGGLE_SWITCH");
+
+		ClickNoError.clickNoMouseOver(
+			key_webContentTitle = "Add Account Group Relation to Simple T-Shirt",
+			locator1 = "Button#BUTTON_PLUS_PRODUCT_VISIBILITY");
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase:109]: i think is possible to use: `Click(locator1 = "Button#PLUS")`

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path [hunk 2]
```diff
@@ -858,6 +858,11 @@
 	<td>//button[contains(@class,'ytp-large-play-button')]</td>
 	<td></td>
 </tr>
+<tr>
+    <td>BUTTON_PLUS_PRODUCT_VISIBILITY</td>
+    <td>//li[contains(@class,'nav-item')]//button[contains(@title,'${key_webContentTitle}')]</td>
+    <td></td>
+</tr>
```
Comment (Pietro69) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path:865]: I believe you can remove it

**Review by Pietro69 — CHANGES_REQUESTED** [review]

**Comment by Pietro69** [comment]
Please add the `property test.name.skip.portal.instance = "CPCommerceProductVisibility#RemoveAccountGroupFilter"`

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase [hunk 3]
```diff
@@ -57,12 +57,82 @@ definition {
...
+		CommerceEntry.addNewCatalogsEntry(
+			catalogCurrency = "US Dollar",
+			catalogLanguage = "English (United States)",
+			catalogName = "Master");
+
+		AssertClickNoError(
+			locator1 = "Button#SUBMIT",
+			value1 = "Submit");
+
+		Refresh();
```
[the remark this answers is not in the document]
Comment (AfonsoHSantos) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase:79]: fixed

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase [hunk 4]
```diff
@@ -57,12 +57,82 @@ definition {
...
+		CPCommerceCatalog.newProductsEntry(
+			productCatalog = "Master",
+			productName = "Simple T-Shirt",
+			productType = "Simple");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Visibility");
+
+		Click(
+			key_toggleSwitch = "wiwk___toggle-switch",
+			locator1 = "ToggleSwitch#SPECIFIC_TOGGLE_SWITCH");
```
[the remark this answers is not in the document]
Comment (AfonsoHSantos) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase:105]: fixed

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase [hunk 5]
```diff
@@ -57,12 +57,82 @@ definition {
...
+
+		Button.click(button = "Delete");
+
+		SelectFrameTop();
+
+		AssertElementPresent(locator1 = "Message#SUCCESS");
+
+		Click(
+			key_toggleSwitch = "wiwk___toggle-switch",
+			locator1 = "ToggleSwitch#SPECIFIC_TOGGLE_SWITCH");
```
[the remark this answers is not in the document]
Comment (AfonsoHSantos) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase:133]: fixed

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase [hunk 6]
```diff
@@ -57,12 +57,82 @@ definition {
...
+
+		ClickNoError.clickNoMouseOver(
+			key_webContentTitle = "Add Account Group Relation to Simple T-Shirt",
+			locator1 = "Button#BUTTON_PLUS_PRODUCT_VISIBILITY");
+
+		SelectFrame.selectFrameNoLoading(locator1 = "IFrame#MODAL_BODY");
+
+		Check(
+			key_checkboxName = "Guest",
+			locator1 = "Checkbox#ENABLE_CHECKBOX_SELECT_CHANNEL");
```
[the remark this answers is not in the document]
Comment (AfonsoHSantos) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase:115]: the change in #903 didn't work, so I didn't change it.

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/ToggleSwitch.path [hunk 7]
```diff
@@ -58,6 +58,11 @@
 	<td>//label[contains(.,'${key_checkSetupName}')]//input[contains(@class,'field') and contains(@class,'toggle-switch')]</td>
 	<td></td>
 </tr>
+<tr>
+    <td>SPECIFIC_TOGGLE_SWITCH</td>
+    <td>//span[contains(@id,'${key_toggleSwitch}')]</td>
+    <td></td>
+</tr>
```
[the remark this answers is not in the document]
Comment (AfonsoHSantos) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/ToggleSwitch.path:65]: fixed

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Checkbox.path [hunk 8]
```diff
@@ -498,6 +498,11 @@
 	<td>//label[contains(.,'${key_checkboxOption}')]/input[@type='checkbox']/../span</td>
 	<td></td>
 </tr>
+<tr>
+    <td>ENABLE_CHECKBOX_SELECT_CHANNEL</td>
+    <td>//tr[contains(@data-name,'${key_checkboxName}')]//input[contains(@type,'checkbox')]</td>
+    <td></td>
+</tr>
```
[the remark this answers is not in the document]
Comment (AfonsoHSantos) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Checkbox.path:505]: fixed

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase (hunk 1 above)
Comment (AfonsoHSantos) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase:109]: isn't possible.

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path (hunk 2 above)
Comment (AfonsoHSantos) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path:865]: how the previous change didn't work, I kept.

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by iovaneale** [comment]
ci:test:commerce-functional

**Comment by iovaneale** [comment]
ci:forward

**Pull request closed without merging**